bigtable/google/cloud/bigtable/table.py

```diff
@@ -184,6 +184,16 @@ def create(self):
                                                 table_id=self.table_id,
                                                 table={})
 
+    def exist(self):
+        """Checks whether a table with given table ID exist
+        or not for given Bigtable instance.
+        """
```

Comment:
A cleaner / faster implementation is to use the table admin API `get_table` method, e.g.:

```python
        table_client = self._instance._client.table_admin_client
        instance_name = self._instance.name
        try:
            table_client.get_table(name=self.name)
        except exceptions.NotFound:
            return False
        else:
            return True
```